WordPress/src/main/java/org/wordpress/android/ui/reader/utils/ReaderImageScanner.java

```diff
@@ -105,8 +105,21 @@ public String getLargestImage(int minImageWidth) {
             if (width > currentMaxWidth) {
                 currentImageUrl = imageUrl;
                 currentMaxWidth = width;
-            } else if (currentImageUrl == null && hasSuitableClassForFeaturedImage(imageTag)) {
-                currentImageUrl = imageUrl;
+            } else {
+                // Inspect the image tag and look for clues for an image of the right size
```

Comment:
I think it makes sense for the two width-less checks (`hasSuitableClassForFeaturedImage` and `getLargeFileAttr`) not to update the `currentMaxWidth`, since there's we don't know the actual width of the photos in those cases (this is also how `hasSuitableClassForFeaturedImage` already worked I believe). Let me know if you think otherwise.

However the `srcset` check can have that information, so it updates the `currentMaxWidth` as of 7a5c5cf.

Reply:
Perfect, thanks!